Source/Scripts/upload-symbols.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
TODO BG, extract this to a submodule?

Reply:
Here `upload-symbols.sh` script is different from the one that is provided along with the native iOS SDK, so I'm not sure whether it's worth extracting it to a submodule.